dialog/file_test.go

```diff
@@ -152,13 +203,21 @@ func TestShowFileOpen(t *testing.T) {
 	// This will only execute if we have a file in the home path.
 	// Until we have a way to set the directory of an open file dialog.
 	test.Tap(target)
-	assert.Equal(t, filepath.Base(target.path), nameLabel.Text)
+	assert.Equal(t, target.location.Name(), nameLabel.Text)
 	assert.False(t, open.Disabled())
 
 	test.Tap(open)
 	assert.Nil(t, win.Canvas().Overlays().Top())
 	assert.Nil(t, openErr)
-	assert.Equal(t, "file://"+target.path, chosen.URI().String())
+
+	targetLocationString := target.location.String()
```

Comment:
This should not be needed as you are comparing a URI with a URI

Reply:
I changed it back since now all URIs just have ```/```